cepgen-toolfile.spec

```diff
@@ -0,0 +1,33 @@
+### RPM external cepgen 1.2.1patch1_gcc700
```

Comment:
@forthommel , please check the version to `1.0` here. This version is for the spec file itself. The generated cepgen.xml will still contain the correct version (@TOOL_VERSION@ will be replaced by the actual version of cepgen).